src/Core.UnitTests/Helpers/WebServiceHelperTests.cs

```diff
@@ -46,102 +46,103 @@ async public Task CallSucceeds()
         }
 
         [TestMethod]
-        public void HttpRequestException_NoInnerException_IsHandledAndLogged()
+        public async Task HttpRequestException_NoInnerException_IsHandledAndLogged()
         {
             // Arrange
             var testLogger = new TestLogger();
 
             // Act
-            WebServiceHelper.SafeServiceCallAsync<bool>(() => throw new HttpRequestException("outer message"),
-                testLogger).Wait();
```

Comment:
We shouldn't need to use `Wait()` in tests - we can just make the test `async Task` and `await`.